packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -972,5 +972,28 @@
   ],
   "cloud-security-posture-settings": [
     "rules"
+  ],
+  "elastic-ai-assistant-prompts": [
+    "content",
+    "created_at",
+    "created_by",
+    "id",
+    "is_default",
+    "is_new_conversation_default",
+    "is_shared",
+    "name",
+    "prompt_type",
+    "updated_at",
+    "updated_by"
+  ],
+  "elastic-ai-assistant-anonimization-fields": [
+    "created_at",
+    "created_by",
+    "default_allow",
+    "default_allow_replacement",
+    "field_id",
```

Comment:
nit: Consider `field` if `field_id` represents an ES field e.g. `user.name` for consistency with other entries in `current_mappings.json` (that appear to also represent ES fields), e.g.:

```json
          "field": {
            "type": "keyword"
          },
```